docs/release/fundamentals/tables-and-data-operations.ipynb

```diff
@@ -0,0 +1,2565 @@
+       "      <th id=\"T_8e80a_level0_col2\" class=\"col_heading level0 col2\" >Computed With</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <td id=\"T_8e80a_row0_col0\" class=\"data row0 col0\" >film_name</td>\n",
+       "      <td id=\"T_8e80a_row0_col1\" class=\"data row0 col1\" >string</td>\n",
+       "      <td id=\"T_8e80a_row0_col2\" class=\"data row0 col2\" ></td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td id=\"T_8e80a_row1_col0\" class=\"data row1 col0\" >year</td>\n",
+       "      <td id=\"T_8e80a_row1_col1\" class=\"data row1 col1\" >int</td>\n",
+       "      <td id=\"T_8e80a_row1_col2\" class=\"data row1 col2\" ></td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td id=\"T_8e80a_row2_col0\" class=\"data row2 col0\" >revenue</td>\n",
+       "      <td id=\"T_8e80a_row2_col1\" class=\"data row2 col1\" >float</td>\n",
+       "      <td id=\"T_8e80a_row2_col2\" class=\"data row2 col2\" ></td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n"
+      ],
+      "text/plain": [
+       "<pandas.io.formats.style.Styler at 0x1732fc310>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "films_t.describe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Besides the column names and types, there's a third element to the schema, `Computed With`. We'll explain what this means in the next section of the tutorial, [Computed Columns](...)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "All of the methods we've discussed so far, such as `insert()` and `get_table()`, are documented in the [Pixeltable API](https://pixeltable.github.io/pixeltable/) Documentation. The following pages are particularly relevant to this section of the tutorial:\n",
+    "- [pixeltable](https://pixeltable.github.io/pixeltable/api/pixeltable/) package reference\n",
+    "- [pxt.Table](https://pixeltable.github.io/pixeltable/api/table/) class reference\n",
+    "- [API Cheat Sheet](https://pixeltable.github.io/pixeltable/api-cheat-sheet/)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### A Real-World Example: Earthquake Data\n",
+    "\n",
+    "Now let's dive a little deeper into Pixeltable's data operations. To showcase all the features, it'll be helpful to have a real-world dataset, rather than our toy dataset with four movies. The dataset we'll be using consists of Earthquake data drawn from the US Geological Survey: all recorded Earthquakes that occurred within 100 km of Seattle, Washington, between January 1, 2023 and June 30, 2024.\n",
+    "\n",
+    "The dataset is in CSV format, and we can easily load it into Pixeltable with the handy `import_csv()` function, which creates a new Pixeltable table from the contents of a CSV file."
```

Comment:
I would leave out terms like "easily", which has a slight marketing ring to it (and doesn't add anything - the user can decide how easy it is).